.github/workflows/bazel.yml

```diff
@@ -0,0 +1,26 @@
+name: Build and Test with Bazel
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    timeout-minutes: 120
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+
+    runs-on: ${{matrix.os}}
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Download dependencies
+        run: python3 utils/git-sync-deps
```

Comment:
FYI I added a --treeless option when downloads are only used once.

But I think we can also cache the external/ directory to save downloads.
And in that case you *don't* want --treeless.